pkgs/development/libraries/fox/default.nix

```diff
@@ -1,31 +1,32 @@
 { lib
 , stdenv
 , fetchurl
+, fetchpatch
 , libpng
 , libjpeg
 , libtiff
 , zlib
 , bzip2
+, libGL
+, libGLU
 , libXcursor
 , libXext
 , libXrandr
 , libXft
-, CoreServices ? null
+, CoreServices
 }:
 
 stdenv.mkDerivation rec {
   pname = "fox";
-  version = "1.7.9";
+  version = "1.7.81";
 
   src = fetchurl {
-    url = "ftp://ftp.fox-toolkit.org/pub/${pname}-${version}.tar.gz";
-    sha256 = "1jb9368xsin3ppdf6979n5s7in3s9klbxqbwcp0z8misjixl7nzg";
+    url = "http://fox-toolkit.org/ftp/${pname}-${version}.tar.gz";
```

Comment:
no https in 2022...

Reply:
Their SSL certificate is self-signed, so no-go...